geodiff/src/geodiff.cpp

```diff
@@ -694,7 +694,7 @@ int GEODIFF_makeCopySqlite( const char *src, const char *dst )
   Sqlite3Db dbFrom, dbTo;
   try
   {
-    dbFrom.openReadOnly( src );
```

Comment:
do we need this function? or add docs to the function?

Reply:
ok let me get rid of that function...